packages/circuit-ui/components/Table/Table.tsx

```diff
@@ -207,7 +207,7 @@ type TableState = {
 class Table extends Component<TableProps, TableState> {
   state: TableState = {
     sortedRow: undefined,
-    sortedRows: undefined,
+    rows: undefined,
```

Comment:
This needs to be initialized from props, otherwise, the rows aren't shown on the initial render with breaks SSR and can cause other errors.